examples/baselines/decision-transformer/decision_transformer/Eval.py

```diff
@@ -0,0 +1,115 @@
+import torch
+import numpy as np
+
```

Comment:
can you make sure to run the code formatter? 

```
pip install isort black
```

Then run

```
isort mani_skill2
black -l mani_skill2
```

to auto format the code (we should add pre-commit hooks later.)